src/main/kotlin/com/sparetimedevs/consonance/App.kt

```diff
@@ -62,9 +55,9 @@ fun Application.module() {
     }
 }
 
-fun main() {
-    val port = Integer.valueOf(System.getenv("PORT"))
-    Dependency.load()
+fun main(args: Array<String>) {
+    val portEnv = args[0]
```

Comment:
Seems like this could be gotten from some other local source instead of a command line argument.

Reply:
Removed.